src/generate_pisa_index.cpp

```diff
@@ -155,7 +133,6 @@ int main(int argc, char const *argv[])
     google::protobuf::io::ZeroCopyInputStream* postings_stream = new google::protobuf::io::IstreamInputStream(&postings_data);
```

Comment:
RAW POINTER :warning: :warning: :warning:

Why do we even store it in a different type than the one we're creating? Is this how it must be done? Couldn't we do the following?
```cpp
google::protobuf::io::IstreamInputStream posting_stream(&postings_data);
google::protobuf::io::CodedInputStream coded_stream(&postings_stream);
```
If not, then can we do at least
```
std::unique_ptr<google::protobuf::io::ZeroCopyInputStream> posting_stream = std::make_unique<google::protobuf::io::IstreamInputStream>(&posting_data);
```
I think this should work but not 100% sure.